source/features/hide-inactive-deployments.tsx

```diff
@@ -2,11 +2,14 @@ import select from 'select-dom';
 import features from '../libs/features';
 
 function init(): void {
-	const deployments = select.all('.TimelineItem .deployment-meta');
+	const deployments = select.all(`
+		.TimelineItem [title^="Deployment Status Label:"],
+		.js-socket-channel[data-url$="/pull_requests/events/deployed"] .TimelineItem a.btn
+	`);
 	deployments.pop(); // Don't hide the last deployment, even if it is inactive
 
 	for (const deployment of deployments) {
-		if (select.exists('.is-inactive', deployment)) {
+		if (deployment.title === 'Deployment Status Label: Inactive') {
```

Comment:
This should be reset to what it was
```suggestion
 		if (select.exists('.is-inactive', deployment)) {
```

Reply:
It's actually a different selector (`[title="Deployment Status Label: Inactive"]`), as that was the change that triggered this PR :wink:

I get the idea though.